dev-libs/ocl-icd/ocl-icd-2.3.0.ebuild

```diff
@@ -15,7 +15,7 @@ KEYWORDS="amd64 x86"
 
 # Does nothing now but by keeping it here we avoid having to have virtual/opencl
 # handle ebuilds both with and without this flag.
-IUSE="+khronos-headers"
+IUSE="+khronos-headers prefix"
```

Comment:
Do we need the `prefix.eclass`?

Reply:
No we don't. We only have to check whether `prefix` USE flag is on, and use `${EPREFIX}` variable is currently enough

Reply:
I don't think you need to append `prefix` to `IUSE` either.  It is covered by IUSE_IMPLICIT.

see `profiles/base/make.defaults:8` and PMS 11.1.1 for details.

Reply:
You're right. Now I have removed that use flag.